iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -297,6 +297,9 @@ private void updateHmsTableProperties(org.apache.hadoop.hive.metastore.api.Table
           "Table location not set");
     }
 
+    // Remove null values from hms table properties
+    hmsTable.getParameters().entrySet().removeIf(e -> e.getValue() == null);
```

Comment:
Do we need to remove null keys here too, just like in `getCatalogProperties`?

Reply:
Right, changed it.